### PR TITLE
feat(Implemented-Search-for-the-frontend): Implemented the search for…

### PR DESCRIPTION
**Pull request by ShankYadav**
… the frontend and its connection with the unbxdApi which returns searched results for the search

### CODE REVIEW

**Comment by yashwal** [comment]
[API to retrieve product details from database](https://trello.com/c/8k0Re7NP/6-api-to-retrieve-product-details-from-database)

**Comment by ShankYadav** [comment]
[Skeletal frontend using HTML and CSS](https://trello.com/c/4Mq6Q2VS/5-skeletal-frontend-using-html-and-css)

**Comment by ShankYadav** [comment]
[Skeletal frontend using HTML and CSS](https://trello.com/c/4Mq6Q2VS/5-skeletal-frontend-using-html-and-css)

File: frontend/header.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+function getQuery(){
+    let searchQuery = document.getElementById("query").value
+    console.log(searchQuery)
```
Comment (priyal-unbxd) [inline frontend/header.js:3]: Remove console logs

**Pull request merged**